test/test_dependencies.jl

```diff
@@ -0,0 +1,54 @@
+using Mimi
+
+Pkg.add("InfoZIP")
+
+#list of URLs of branches of packages to test
+dependencies = [
+"https://github.com/davidanthoff/fund.jl/archive/master.zip"
+]
+
+#list of failed tests to build as you go
+errors = []
+#make a temporary directory to run the tests in
+tmp_path = joinpath(dirname(@__FILE__),"tmp_testing/")
```

Comment:
Minor nitpick, I think you can leave out the ``/`` at the end, that is platform specific and ``joinpath`` should take care of that in any case.